src/gui/screen_menu_experimental_settings_mini.cpp

```diff
@@ -21,8 +21,8 @@
 using Screen = ScreenMenu<EHeader::Off, EFooter::On, HelpLines_None, MI_SAVE_AND_RETURN,
     MI_Z_AXIS_LEN, MI_RESET_Z_AXIS_LEN, MI_STEPS_PER_UNIT_E, MI_RESET_STEPS_PER_UNIT, MI_DIRECTION_E, MI_RESET_DIRECTION>;
 class ScreenMenuExperimentalSettings : public Screen {
-    static constexpr const char *const save_and_reboot = N_("Do you want to save changes and reboot the printer?");
```

Comment:
Why did you removed experimental menu translations?
It is accessible in release too - via long press.

Reply:
Yes, it is, but we agreed the experimental menu will not be translated.